data/setup.xml

```diff
@@ -4,13 +4,13 @@
 	</setup>
 	<setup key="Audio" title="Audio Settings" level="1" showOpenWebif="1">
 		<item level="1" text="PCM Multichannel" description="Choose whether multi channel sound tracks should be output as PCM." requires="CanPcmMultichannel">config.av.pcm_multichannel</item>
-		<item level="1" text="AC3 downmix" description="Choose whether AC3 sound tracks should be downmixed to stereo." requires="CanDownmixAC3">config.av.downmix_ac3</item>
+		<item level="1" text="AC3 down mix" description="Choose whether AC3 sound tracks should be down mixed to stereo." requires="CanDownmixAC3">config.av.downmix_ac3</item>
```

Comment:
downmix is a technical word and is correct.  That is, "downmix" and "downmixed" are correct.